### PR TITLE
'edname' key in specials is now 'key'; add 'fdname' key

### PR DESCRIPTION
**Pull request by felixlinker**
This pr targets two issues:
1. It makes clear what the actual name of special effects is; and it does so by using the same key as blueprints (`"fdname"`)
2. It takes the very confusing key `"edname"` which sounds like it means the same thing as `"fdname"` and renames it to `"key"` which makes it very clear what this is; a redundant value that is used by coriolis when working with special effects

This confused me a lot when I was fixing some special effects. However, I am not clear what the impact might be if this changes goes to master.

### CODE REVIEW

**Comment by felixlinker** [comment]
Before this PR gets merged:

- [ ] Also refactor `fdname` key for blueprints

File: modules/hardpoints/ax_missile_rack.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     {
       "experimental": true,
       "ammo": 64,
+      "ammocost": 235,
```
Comment (willyb321) [inline modules/hardpoints/ax_missile_rack.json:6]: Is this for a full reload or 1 clip?

Comment (felixlinker) [reply modules/hardpoints/ax_missile_rack.json:6]: It is a single shot. This was already present for AFNMU, Chaff and Heatsinks.

**Comment by felixlinker** [comment]
Thanks for the review. I think, I will simply continue to rewrite coriolis to ed-forge and keep this branch as a dev branch. So let's not merge it for now.

File: modules/hardpoints/ax_missile_rack.json (hunk 1 above)
Comment (willyb321) [reply modules/hardpoints/ax_missile_rack.json:6]: awesome thanks for clearing that up

**Comment by willyb321** [comment]
Yeah I guess that would be the best course of action for now.

**Comment by felixlinker** [comment]
Closing pull-request as this branch will be used for the ed-forge update eventually.

**Pull request closed without merging**